File: 1984.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var numero = lines.shift().trim();
var tamanho_numero = numero.length - 1;
var string = "";

for (var i = tamanho_numero; i >= 0; i--) {
    string += numero[i];
}

console.log(string);

File: 1984.c
#include <stdio.h>
#include <string.h>
 
int main() {
    char numero[11];
    scanf("%s", numero);
    int tamanho_numero = strlen(numero);
    for (int i = tamanho_numero - 1; i >= 0; i--) {
        printf("%c", numero[i]);
    }
    printf("\n");
 
    return 0;
}
